tests/tests/lib.py

```diff
@@ -45,80 +60,117 @@ def run_command_with_agent(cmd: str) -> bytes:
+    Read the contents of the given filename to a list of strings. If it is a
+    normal file, uses the standard open() function. Otherwise, uses `keybase fs
+    read`. This is because fuse is not running in the container so a normal
+    open call will not work for KBFS.
     :param filename:    The name of the file to read
     :return:            A list of lines in the file
     """
     if filename.startswith("/keybase/"):
         return run_command(f"keybase fs read {filename}").splitlines()
-    with open(filename, 'rb') as f:
+    with open(filename, "rb") as f:
         return f.readlines()
 
+
 def clear_keys():
     # Clear all keys generated by kssh
     try:
         run_command("rm -rf ~/.ssh/keybase-signed-key*")
     except subprocess.CalledProcessError:
         pass
 
+
 def clear_local_config():
     # Clear kssh's local config file
     try:
         run_command("rm -rf ~/.ssh/kssh-config.json")
     except subprocess.CalledProcessError:
         pass
 
+
 def load_env(filename: str):
-    # Load the environment based off of the given filename which is the path to the python test script
+    # Load the environment based off of the given filename which is the path to
+    # the python test script
     env_name = os.path.basename(filename).split(".")[0]
-    return requests.get(f"http://ca-bot:8080/load_env?filename={env_name}").content == b"OK"
+    return (
+        requests.get(f"http://ca-bot:8080/load_env?filename={env_name}").content
+        == b"OK"
+    )
+
 
 def assert_contains_hash(expected_hash: bytes, output: bytes):
     assert expected_hash in output
 
+
 @contextmanager
 def simulate_two_teams(tc: TestConfig):
-    # A context manager that simulates running the given function in an environment with two teams set up
-    run_command(f"keybase fs read /keybase/team/{tc.subteam}.ssh.staging/kssh-client.config | "
-                     f"sed 's/{tc.subteam}.ssh.staging/{tc.subteam_secondary}/g' | "
-                     f"sed 's/{tc.bot_username}/otherbotname/g' | "
-                     f"keybase fs write /keybase/team/{tc.subteam_secondary}/kssh-client.config")
+    # A context manager that simulates running the given function in an
+    # environment with two teams set up
+    run_command(
+        f'echo \'{{"method": "get", "params": {{"options": {{"team": '
+        f'"{tc.subteam}.ssh.staging", "namespace": "__sshca", '
+        f'"entryKey": "kssh_config"}}}}}}\' | '
+        f"xargs -0 -I get keybase kvstore api -m get | "
+        f"jq '.result' | "
+        f"sed 's/{tc.subteam}.ssh.staging/{tc.subteam_secondary}/g' | "
+        f"sed 's/{tc.bot_username}/otherbotname/g' | "
+        f"jq '. | "
+        f'{{"method":"put", "params": {{"options": {{"team": '
+        f'.teamName, "namespace": .namespace, "entryKey": .entryKey,'
+        f'"entryValue":.entryValue}}}}}}\' | '
+        f"xargs -0 -I put keybase kvstore api -m put"
```

Comment:
this is quite a bit harder to read than what was here before. if we can't figure out a way to make the kvstore json commands look a lot nicer, how about just two `run_command` calls. fetch, update in python-land, put.